source/_integrations/kaleidescape.markdown

```diff
@@ -0,0 +1,136 @@
+---
+title: Kaleidescape
+description: Instructions on how to integrate Kaleidescape into Home Assistant.
+ha_category:
+  - Media Player
+ha_release: '2022.4'
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_ssdp: true
+ha_codeowners:
+  - '@SteveEasley'
+ha_domain: kaleidescape
+ha_platforms:
+  - media_player
+---
+
+The Kaleidescape integration allows for the automation of Kaleidescape movie players.
+
+![Screenshot](/images/integrations/kaleidescape/media_player.png)
+
```

Comment:
The frontend card will likely change over time so we shouldn't include it here since it adds maint burden.

Reply:
Fair enough. Remove.